Docs/developer_guide/build_instructions/macos.md

```diff
@@ -176,6 +176,28 @@ Library not loaded: /usr/local/opt/zstd/lib/libzstd.1.dylib
 It means that `libarchive` has has found homebrew versions of some of it's requirements, rather than local ones. For the packaged version of Slicer to run on other machines none of the prerequisites should be installed via homebrew. For example `lz4` and `zstd` are bundled with `subversion` and `rsync` so if you have these two application installed via homebrew, `libarchive` will grab them from `/usr/local/opt/` and the packaged Slicer will not run on other machines. The solution is either to remove them from from homebrew with `brew remove lz4` and `brew remove zsdt` or to change the `$PATH` so that the local build folder goes before `/usr/local/opt/`. After doing this Slicer should be rebuilt and repackaged.
 See [Relevant issue that's tracking this error](https://github.com/Slicer/Slicer/issues/5415)
 
+## Homebrew package manager
+
+Slicer can be installed with a single command using [Homebrew](https://brew.sh/), as described in the [installation documentation](../../user_guide/getting_started.md#installing-using-homebrew).
+
+Specifically, the `cask` extension is used, which allows management of graphical applications from the command line.
+Casks are Ruby files (`.rb`) describing the metadata necessary to download, check and install the package.
+
+The [cask for the Preview version of Slicer](https://github.com/Homebrew/homebrew-cask-versions/blob/master/Casks/slicer-preview.rb) does not need maintenance, as it automatically downloads the latest binaries and therefore the exact version does not need to be specified in the cask.
+However, the [cask for the stable version](https://github.com/Homebrew/homebrew-cask/blob/master/Casks/slicer.rb), which has traditionally been [maintained by Slicer users](https://github.com/Homebrew/homebrew-cask/commits/master/Casks/slicer.rb), needs to be manually updated every time a new stable version of Slicer is released.
+Instructions to update Homebrew casks can be found on the [`homebrew-cask` repository](https://github.com/Homebrew/homebrew-cask/commits/master/Casks/slicer.rb).
```

Comment:
Is [this](https://github.com/Homebrew/homebrew-cask/blob/HEAD/CONTRIBUTING.md#updating-a-cask) the instruction you are referring to?

How do you determine the version number `4.11.20210226,1442768`?
Where does the `1442768` come from?

Reply:
> Is this the instruction you are referring to?

Yes, thanks. That was a bad paste.

> Where does the 1442768 come from?

I suppose that the [bitstream ID](https://discourse.slicer.org/t/slicer-installation-on-mac-using-homebrew/933/19).

> How do you determine the version number 4.11.20210226,1442768?

I just look at the version on the download website and at its download link (https://download.slicer.org/bitstream/1442768). @jcfr provided in the link above a method to retrieve these automatically, if I understood correctly.